llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -5690,6 +5690,42 @@ Value *SPIRVToLLVM::transSPIRVImageAtomicOpFromInst(SPIRVInstruction *bi, BasicB
   return result;
 }
 
+// =============================================================================
+// Helper function for handling converting sampler select ladder
+//
+// @param result : [in] The result from non-converting sampler path.
+// @param convertingSamplerIdx : [in] The index of converting sampler that will be used in alternativeFunc.
+// @param alternativeFunc : [in] A function that accepts converting sampler and returns the same type as result.
```

Comment:
`alternativeFunc` doesn't sound descriptive to me. How about `createImageOp`? Both callers use lambdas that are named `createImageXxx`, so it fits. I don't think we need the `Func` suffix necessarily.

Reply:
Yes, that sounds make sense, replaced.